src/pipeline.js

```diff
@@ -94,6 +94,37 @@ class Pipeline {
     this._posts = [];
     // functions that are executed before each step
     this._taps = [];
+
+    this.attach = function attach(ext) {
+      if (this.sealed) {
+        return;
```

Comment:
I'd throw an error instead, as this might indicate a developer error…

Reply:
No, running the pipeline twice is ok. I just don't want to re-add the extension points.